guides/v2.1/cloud/live/stage-prod-test.md

```diff
@@ -203,7 +203,7 @@ If you encounter issues, save your reproduction steps, error messages, strange s
 
 Before launching, we highly recommend performing extensive traffic and performance testing on your Staging and Production environments.  You should consider performance testing for your frontend and backend processes.
 
-Before you begin testing, please enter a ticket with support advising the environments you are testing, what tools you are using, and the time frame. Update the ticket with results and information to track performance. When you complete testing, add your updated results and note in the ticket testing is complete with a date and time stamp.
+Before you begin testing, enter a ticket with support advising the environments you are testing, what tools you are using, and the time frame. Update the ticket with results and information to track performance. When you complete testing, add your updated results and note in the ticket testing is complete with a date and time stamp.
```

Comment:
advising them about the ....  the test tools you are using, and the testing time frame

Reply:
Some other time. I didn't read much.